lib/engine/game/g_18_cz.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require_relative '../config/game/g_18_cz'
+require_relative 'base'
+
+module Engine
+  module Game
+    class G18CZ < Base
+      register_colors(brightGreen: '#c2ce33', beige: '#e5d19e', lightBlue: '#1EA2D6', mintGreen: '#B1CEC7',
+                      yellow: '#ffe600', lightRed: '#F3B1B3')
+
+      load_from_json(Config::Game::G18CZ::JSON)
+
+      DEV_STAGE = :prealpha
+      GAME_LOCATION = 'Czech Republic'
+      GAME_RULES_URL = 'https://www.lonny.at/app/download/9940504884/rules_English.pdf'
+      GAME_DESIGNER = 'Leonhard Orgler'
+      GAME_PUBLISHER = nil
+      GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/18CZ'
+
+      SELL_BUY_ORDER = :sell_buy
+      SELL_MOVEMENT = :down_block
+
+      HOME_TOKEN_TIMING = :operate
+
+      PAR_RANGE = {
+        1 => [50, 55, 60, 65, 70],
+        2 => [60, 70, 880, 90, 100],
+        3 => [90, 100, 110, 120],
+
+      }.freeze
+
+      COMPANY_VALUES = [40, 45, 50, 55, 60, 65, 70, 75, 80, 90, 100, 110, 120].freeze
+
+      LAYER_BY_NAME = {
+        'SX' => 3,
+        'BY' => 3,
+        'PR' => 3,
+        'kk' => 3,
+        'Ug' => 3,
+        'BN' => 2,
+        'NWB' => 2,
+        'ATE' => 2,
+        'BTE' => 2,
+        'KFN' => 2,
+        'EKJ' => 1,
+        'OFE' => 1,
+        'BCB' => 1,
+        'MW' => 1,
+        'VBW' => 1,
+      }.freeze
+
+      EVENTS_TEXT = Base::EVENTS_TEXT.merge(
```

Comment:
Could possibly be done as STATUS instead. EVENT is more .... events. While STATUS is for status. But both work.

Reply:
I think these should be events since they have triggers

https://github.com/tobymao/18xx/blob/f7222bd1611a80c8d0921197f2bf0bda42da96ab/lib/engine/game/g_18_cz.rb#L140